common/lib/xmodule/xmodule/poll_module.py

```diff
@@ -195,9 +196,10 @@ def definition_to_xml(self, resource_fs):
         xml_object.set('display_name', self.display_name)
 
         def add_child(xml_obj, answer):
+            answer_text = str(Text(answer['text']))  # escaped characters.
```

Comment:
nit: I would say add a more comprehensive comment here as to why we are escaping here, may be something like :

```
# Escape answer text before adding to xml tree. 
answer_text = str(Text(answer['text']))
```